ftp/src/main/scala/akka/stream/alpakka/ftp/model.scala

```diff
@@ -74,6 +77,9 @@ final case class FtpSettings(
 
   def withPassiveMode(passiveMode: Boolean): FtpSettings =
     copy(passiveMode = passiveMode)
+
+  def withConfigureConnection(configureConnection: FTPClient => Unit): FtpSettings =
```

Comment:
@ennru I'm assuming this is intended mostly for the Java api. Should I make this take a `Function[FTPClient, Unit]`? Or do I need to use `Void`? I never made Java api in Scala before :).

I want to add an example to the docs, but I'm struggling with how to use this in Java 😅

Reply:
It would be [`java.util.function.Consumer`](https://docs.oracle.com/javase/8/docs/api/java/util/function/Consumer.html)

Reply:
Tnx. I was unable to convert that to Scala. Luckily, Java's lambda fit in elegantly.